src/renderer/components/WalletsPage.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class="wallets-page">
+    <div class="sidemenu">
+      <router-link
+        v-for="wallet in wallets"
+        :key="wallet.id"
+        class="sidemenu__item"
+        :to="'/dashboard/wallets-page/' + wallet.id"
+      >
+        <div>{{ wallet.name }}</div>
+        <div>{{ wallet.amount }}</div>
+      </router-link>
+    </div>
+
+    <div class="main">
+      <router-view></router-view>
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'wallets-page',
+
+    data () {
+      return {
+        wallets: [
+          { id: 'wallet-1', name: 'dollar#russia', amount: '100.00' },
+          { id: 'wallet-2', name: 'yen#russia', amount: '100.00' },
+          { id: 'wallet-3', name: 'euro#russia', amount: '100.00' }
+        ]
+      }
+    }
+  }
+</script>
+
+<style lang="scss" scoped>
+  @import "~@/styles/element-variables.scss";
+
+  .wallets-page {
+    display: flex;
+  }
+
+  .sidemenu {
+    $sidemenu-width: 150px;
+
+    &__item {
+      display: block;
+      width: $sidemenu-width;
+      padding: 1rem;
+      text-decoration: none;
+      color: #7e7e7e;
+      background: #9d9d9d;
+      border: 1px solid #aaa;
```

Comment:
Please remove right border `1px solid white` , so it looks like this :)
<img width="321" alt="screen shot 2018-04-24 at 11 31 54" src="https://user-images.githubusercontent.com/10958990/39175485-2038af54-47b3-11e8-88ca-870f6e7d40a5.png">